vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -105,7 +105,7 @@ def create_weights(self, layer: torch.nn.Module, input_size: int,
                 dtype=params_dtype,
             )
         }
-        if self.group_size == -1:
+        if not partition_scales:
```

Comment:
Can you explain this?

Reply:
better condition for when we're in both cases. I think it might end up being the same but more consistent with what ive seen in other quant config files